### DISCUSSION TITLE
[Feature Request]: 支持按演员名搜索资源

### DISCUSSION THREAD

**Discussion by thsrite** [jxxghp/MoviePilot#3384] [讨论]
### 当前程序版本

v1.3.7

### 运行环境

Docker

### 功能改进类型

主程序

### 功能改进

![image](https://github.com/jxxghp/MoviePilot/assets/54088512/f9560f7b-05fd-4717-9297-ad12d23906b0)
除了可直接搜索电影电视剧外，可否按照演职人员名称搜索该人员参演的作品

**Comment by Shurelol** [comment]
@thsrite 之前nt那个功能么
